pkg/gencmd/context.go

```diff
@@ -0,0 +1,32 @@
+package gencmd
+
+import (
+	"github.com/izumin5210/grapi/pkg/grapicmd"
+	"github.com/pkg/errors"
+)
+
+func defaultCtx() *Ctx {
+	return &Ctx{
+		Ctx: &grapicmd.Ctx{},
+	}
+}
+
+type Ctx struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Ctx should have comment or be unexported